app/domain/authentication/authn_jwt/jwt_configuration_dummy_vendor.rb

```diff
@@ -2,16 +2,16 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class JWTConfigurationDummyVendor < JWTConfigurationInterface
-      def get_identity
-        return "cucumber"
+      def get_conjur_id
+        "cucumber"
       end
 
       def validate_restrictions
-        return true
+        true
```

Comment:
`validate` method should throw if failed, so it should not return value.
```suggestion
```

Reply:
also relevant for the `validate_and_decode_token` method.

Reply:
Will happen out of scope of this PR when real validate_and_decode token function will be implimented